tests/parsec-cli-tests.sh

```diff
@@ -215,6 +217,30 @@ test_csr() {
     delete_key $1 $KEY
 }
 
+test_rsa_key_default_bits() {
+    KEY="anta-key-rsa-default-bits"
+    create_key "RSA" $KEY
+    run_cmd $PARSEC_TOOL_CMD export-public-key --key-name $KEY >${MY_TMP}/checksize-${KEY}.pem
+    run_cmd $OPENSSL rsa -pubin -text -noout -in ${MY_TMP}/checksize-${KEY}.pem >${MY_TMP}/checksize-${KEY}.txt
+    if ! cat ${MY_TMP}/checksize-${KEY}.txt | grep "RSA Public-Key: (2048 bit)"; then
```

Comment:
unnecessary usage of cat here. You can simplify it:
```
if ! grep -q "RSA Public-Key: (2048 bit)" ${MY_TMP}/checksize-${KEY}.txt; then
```

You don't even need the text file:
```
if ! run_cmd $OPENSSL rsa -pubin -text -noout -in ${MY_TMP}/checksize-${KEY}.pem | grep -q "RSA Public-Key: (2048 bit)"; then
```